src/gui/properties/speedplotview.h

```diff
@@ -29,9 +29,12 @@
 #ifndef SPEEDPLOTVIEW_H
 #define SPEEDPLOTVIEW_H
 
+#ifndef Q_MOC_RUN
```

Comment:
AFAIK, we do not use it with other Boost headers and there are no problems. Or is the problem in this particular header?